app/uk/gov/hmrc/customs/financials/emailthrottler/services/EmailNotificationService.scala

```diff
@@ -20,14 +20,16 @@ import play.api.http.Status
 import play.api.{Logger, LoggerLike}
 import uk.gov.hmrc.customs.financials.emailthrottler.config.AppConfig
 import uk.gov.hmrc.customs.financials.emailthrottler.models.EmailRequest
-import uk.gov.hmrc.http.HttpReads.Implicits._
-import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.http.HttpReads.Implicits
```

Comment:
Update to import uk.gov.hmrc.http.HttpReads.Implicits.* as its generating warning during compilation